subsys/mpsl/Kconfig

```diff
@@ -25,6 +25,108 @@ config MPSL_TIMESLOT_SESSION_COUNT
 	help
 	  Maximum number of timeslot sessions.
 
+DT_COMPAT_NORDIC_NRF21540_GPIO := nordic,nrf21540-fem
+DT_COMPAT_SKYWORKS_SKY66112_11 := skyworks,sky66112-11
+
+config CAN_USE_MPSL_FEM_NRF21540_GPIO
+	bool
+	default $(dt_nodelabel_has_compat,nrf_radio_fem,$(DT_COMPAT_NORDIC_NRF21540_GPIO)) && !BT
+
+config CAN_USE_MPSL_FEM_SKY66112_11
+	bool
+	default $(dt_nodelabel_has_compat,nrf_radio_fem,$(DT_COMPAT_SKYWORKS_SKY66112_11))
+
+config MPSL_FEM
+	bool "Radio front-end module (FEM) support"
+# MPSL FEM is not supported on 53 yet
+	depends on !(SOC_NRF5340_CPUAPP || SOC_NRF5340_CPUNET)
+	default CAN_USE_MPSL_FEM_NRF21540_GPIO || CAN_USE_MPSL_FEM_SKY66112_11
+	help
+	  Controls if front-end module (FEM) it to be configured automatically
+	  when MPSL is initialized. Default value and type of FEM to use
+	  is determined by the devicetree.
+
+if MPSL_FEM
+
+choice MPSL_FEM_CHOICE
+	prompt "Radio front-end module (FEM) type"
+	default MPSL_FEM_NRF21540_GPIO if CAN_USE_MPSL_FEM_NRF21540_GPIO
```

Comment:
Instead of putting the defaults here, order the choices in the order that is preferred, so that when both are supported you end up with default choice being `MPSL_FEM_NRF21540_GPIO ` (which I assume is what we want?)

Reply:
That's right, without these defaults it also works as designed.